src/GraphQL.Client.Serializer.SystemTextJson/ErrorPathConverter.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace GraphQL.Client.Serializer.SystemTextJson
+{
+    public class ErrorPathConverter : JsonConverter<ErrorPath>
+    {
+
+        public override ErrorPath Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            using var doc = JsonDocument.ParseValue(ref reader);
```

Comment:
@rose-a Please read this thread https://github.com/graphql-dotnet/server/pull/319

`JsonDocument.ParseValue` is evil

Reply:
I also recommend you to add benchmarks to see what happens. As you can see, there may be unpleasant surprises.

Reply:
That's used in the MapConverter, too...

Reply:
The difference in performance is huge.